api/proto/metrics/metrics.proto

```diff
@@ -177,12 +177,71 @@ message appCpuMetric {
   uint64 total_ns = 7;                 // like 'total' but in nanoseconds. Overflow may happen after ~40 years without EVE restart
 }
 
-// StorageMetric   At the moment not used
+// StorageDeviceMetrics - contains device statistics from ZFS storage
+message StorageDeviceMetrics {
```

Comment:
most of the comments do not bring any value. For example
```
expandable_size = 5; // expandable dev size
```

I can see that from the variable name that it is the "expandable size", but I have only a faint idea what that means. It would be nice to be able to understand API without spending hours in the code, trying to figure out where does this field come from.

Reply:
Corrected